### DISCUSSION TITLE
Link Embed/Preview

### DISCUSSION THREAD

**Discussion by adampaulberg** [mealie-recipes/mealie#856] [Feature Request]
Before submitting this feature request I have
COMPLETE AND FILL THESE OUT

 Reviewed the documentation and verified that my feature request isn't already a feature.
 Reviewed the existing feature requests and my feature has not been requested.
 Checked the tasks taged issues and verified my feature is not covered

Plese Describe The Problem To Be Solved
When deeplinking a recipe on something like Discord or Slack, it would be nice if the embed showed a picture (if available) and the recipe, instead of a short blerb about Mealie.

![image](https://user-images.githubusercontent.com/4030607/144492669-675b19c5-da11-4ca8-bc2d-b408d2b8eaca.png)

(Optional): Suggest A Solution
Make it something more like
![image](https://user-images.githubusercontent.com/4030607/144492749-e6746be0-c5dd-4ab0-9fd0-0a5ffda4addf.png)

**Comment by hay-kot** [comment]
I've created a task for this for the v1 release

- https://github.com/hay-kot/mealie/issues/861

**Comment by hay-kot** [comment]
Got this working in

- #867
